src/focus.service.ts

```diff
@@ -538,9 +538,11 @@ export class FocusService {
    * Returns if the element can receive focus.
    */
   private isFocusable(el: HTMLElement): boolean {
-    const record = this.registry.find(el);
-    if (record && record.exclude && record.exclude()) {
-      return false;
+    for (let parent = el; parent; parent = parent.parentElement) {
```

Comment:
That seems like considerably bad for performance...as it is already called inside a loop
https://github.com/WatchBeam/arcade-machine/blob/3cb586bf1b357293b82df59919e079356238c9e8/src/focus.service.ts#L622